groups/views/comments.py

```diff
@@ -126,19 +126,19 @@ def extract_uuid_from_email(email, request):
         return match.group('uuid').replace('$', ':')
 
     @staticmethod
-    def create_file_comments(request, user, discussion):
+    def create_file_attachments(request, user, comment):
         """
-        Create any number of file comments from the attachments in this message.
+        Create any number of file attachments from the attachments in this message.
 
         Mailgun provides an entry called `attachment-count` to store the number
         of attachments, then each attachment is a separate entry, `attachment-x` where
         `x` is a number.
         """
         for attachment in request.FILES.values():
-            models.FileComment.objects.create(
+            models.AttachedFile.objects.create(
                 file=attachment,
                 user=user,
-                discussion=discussion,
+                attached_to=comment,
             )
```

Comment:
This could use [`bulk_create`](https://docs.djangoproject.com/en/dev/ref/models/querysets/#bulk-create)